### ISSUE TITLE
Scroll event triggered twice on touch screen scroll

### ISSUE THREAD

**Issue by rynoV** [react-spring/react-use-gesture#78]
When listening for scroll events on the window and scrolling on a mobile device with a touch screen the scroll event will sometimes end and then start again and quickly finish. I think it might have something to do with the way the mobile browser resizes and/or handles overflow scroll on the window.

This is the code I'm using to listen for the scroll events:
```javascript
  const bind = useGesture({
    onScroll     : () => {
      // ...code
    },
    onScrollStart: () => {
       // ...code
    },
    onScrollEnd  : () => {
       // ...code
    },
  }, { domTarget: window })

  useEffect(bind, [bind])
```

### Steps to reproduce:
Clone this repo: [https://github.com/rynoV/calumsieppert.me-v2](https://github.com/rynoV/calumsieppert.me-v2)

Yarn install and run:
`yarn develop-mobile`
(You might need to install the gatsby cli, I'm not sure)

Open ip address given by the command on a mobile device. (I was using an iphone 6 with safari and chrome)

Watch how the scroll events are triggered.

**Comment by dbismut** [comment]
Hey, thanks for reporting the issue, but I'm not sure I can reproduce. Can you please clarify and possibly set a simple example on codesandbox (ideally not using `console.log` to make iPhone debugging easier)?

Just to clarify, depending on the version you're using, Safari iOS doesn't handle `scroll` events the same way. Also, the onScrollEnd event is debounced (there's no native `scrollend` event) with a timeout of 100ms. If for some reason the scroll bounces and doesn't emit events for over 100ms and then starts again, you'll indeed have a end scroll firing, and the start and then quickly finishes as you describe.

**Comment by rynoV** [comment]
Thanks for the quick response! [Here is a codesandbox](https://codesandbox.io/s/react-use-gesture-issue-gh1si) where you should be able to reproduce the behaviour I described.

**Comment by dbismut** [comment]
Hey @rynoV I changed the default timeout from 100ms to 120ms and released 5.2.2 but I'm not sure it helps... Let me try this once more when I get home.

**Comment by rynoV** [comment]
Seems to have fixed it, thanks a lot!
